src/cmake/dependencies.cmake

```diff
@@ -357,7 +357,6 @@ if(NOT USE_SYSTEM_MAGNUM)
       MagnumPlugins::AssimpImporter
       MagnumPlugins::BasisImporter
       MagnumPlugins::GltfImporter
-      MagnumPlugins::StanfordImporter
```

Comment:
> remove StanfordImporter as it doesn't support ASCII ply.

Ah, that's one of my TODOs. I'm not interested in implementing parsing of ASCII PLY files, but it could delegate to Assimp for those.

Where do these come from, btw? Is there some Habitat code that writes them? Might be a candidate for refactor / replacement with builtin Magnum PLY export functionality.

What `StanfordImporter` can do and Assimp can't is import of files with per-vertex semantic IDs. I remember that used to be used quite often, not sure if that's still relevant for current dataasets.

Reply:
Thanks for the thoughts on this topic.
I don't think semantic ids in .ply are necessary for python import, but ASCII ply is an easy format for hand-edited or human-readable mesh data. Agreed we generally don't try to use these in practice.